plugins/coreml/src/coreml/__init__.py

```diff
@@ -47,11 +56,11 @@ def get_input_size(self) -> Tuple[float, float]:
     def detect_once(self, input: Image.Image, settings: Any, src_size, cvss):
         out_dict = self.model.predict({'image': input, 'confidenceThreshold': .2 })
 
-        coordinatesList = out_dict['coordinates']
+        coordinatesList = out_dict['coordinates'].astype(float)
```

Comment:
do we need to type check this first? dont previous coremls return a proper float array?